### DISCUSSION TITLE
With Dagster Cloud Hybrid and the ECS Agent, should the agent or the code locations get upgraded first?

### DISCUSSION THREAD

**Discussion by dagsir** [dagster-io/dagster#17639] [Q&A]
With Dagster Cloud Hybrid using the ECS Agent, is there a preferred order for which gets upgraded first, the agent or the code locations when upgrading minor versions (1.3.x to 1.4.x for example)?

---

*The question was originally asked in [Dagster Slack](https://dagster.slack.com/archives/C02LJ7G0LAZ/p1698939977416999?thread_ts=1698939977.416999&cid=C02LJ7G0LAZ).*

**Comment by shalabhc** [comment]
The upgrade should work in either order. In hybrid, users tend to upgrade code servers more frequently than the agent so I would upgrade the code server first if I had to choose one.
